### PR TITLE
EPMRPP-83020 || Add events for Remove account flow

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: app/src/pages/inside/profilePage/modals/deleteAccountModal/deleteAccountModal.jsx [hunk 1]
```diff
@@ -69,7 +70,13 @@ const DeleteAccount = ({ data }) => {
 
   const onDelete = () => {
     const onSuccess = () => {
-      // todo send 'data' to GA4
+      const checkboxes = Object.keys(data).filter((key) => key !== 'otherReason');
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/modals/deleteAccountModal/deleteAccountModal.jsx:73]: we can assign Object.keys(data) to variable and use it.

File: app/src/components/main/analytics/events/profilePageEvents.js [hunk 2]
```diff
@@ -14,8 +14,47 @@
...
+    ...basicClickEventParametersProfile,
+    element_name: 'continue',
+    modal: 'delete_account',
+  },
+  getDeleteBtnDeleteModalEvent: (checkboxes, otherReason) => ({
+    ...basicClickEventParametersProfile,
+    element_name: 'delete',
+    modal: 'delete_account',
+    status: checkboxes.join('#'),
+    type: otherReason.slice(0, GA_4_FIELD_LIMIT),
```
Comment (Bam6ycha) [inline app/src/components/main/analytics/events/profilePageEvents.js:39]: In the document with events, I can not see the `status` and `condition` fields.
Did you discuss these fields with WA?

File: app/src/pages/inside/profilePage/deleteAccountBlock/deleteAccountBlock.jsx [hunk 3]
```diff
@@ -76,13 +78,16 @@ const ButtonWithTooltip = withTooltip({
   },
 })(Button);
 
-export const DeleteAccountBlock = () => {
+export const DeleteAccountBlock = track()(({ tracking }) => {
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/deleteAccountBlock/deleteAccountBlock.jsx:81]: Could you please explain the reason why you don't use hooks?

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/components/main/analytics/events/profilePageEvents.js (hunk 2 above)
Comment (Vadim73i) [reply app/src/components/main/analytics/events/profilePageEvents.js:39]: Yes, this is Yuliya's requirement

**Review by Bam6ycha — APPROVED** [review]

**Pull request merged**